.cirrus.yml

```diff
@@ -443,8 +443,8 @@ task:
   setup_script:
     - date
     - which flutter
-    - sudo gem install bundler -v 2.0.2 -N
-    - bundle install --system --gemfile=dev/ci/mac/Gemfile
+    - bundle --version
+    - sudo bundle install --system --gemfile=dev/ci/mac/Gemfile
```

Comment:
This is because now the latest Ruby is installed via Homebrew rather then using the system one.